packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/service-integration.routing.js

```diff
@@ -81,7 +81,20 @@ export default /* @ngInject */ ($stateProvider) => {
           serviceIntegrationList,
         ) =>
           kafkaServicesList.filter(
-            (k) => !serviceIntegrationList.find((i) => i.serviceId === k.id),
+            (k) =>
+              !serviceIntegrationList.find(
+                (i) => i.destinationServiceId === k.id,
+              ),
+          ),
```

Comment:
As i can see the naming was there before.
It could be better to rename them to be understandable.

```suggestion
            ({id: kafkaServideId}) =>
              !serviceIntegrationList.find(
                ({destinationServiceId}) => kafkaServideId === destinationServiceId,
              ),
          ),
```

Reply:
Done here: https://github.com/ovh/manager/pull/5749/commits/6b99dec7446758a33b50ddefedf3702d2c5106e1